### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by cassella**
Fix typos in too many files to mention.

### CODE REVIEW

**Comment by cassella** [comment]
Updated to resolve merge conflict with main, and fix further typos.

**Comment by cassella** [comment]
Rebased to main and fixed more typos.

**Comment by cassella** [comment]
Updated to main and fixed one more typo.

**Comment by cassella** [comment]
Updated to main and fixed a bunch more typos.

The last commit renames a method in compiler/dyno, which I haven't learned how to test yet.  The commit to resolution-queries.cpp renames a struct that's private to that file.

File: compiler/dyno/include/chpl/types/ClassType.h [hunk 1]
```diff
@@ -100,7 +100,7 @@ class ClassType final : public Type {
     return basicType_;
   }
 
-  /** Returns the verison this ClassType with the passed decorator */
+  /** Returns the version of this ClassType with the passed decorator */
```
Comment (cassella) [inline compiler/dyno/include/chpl/types/ClassType.h:103]: @mppf Was this what was meant here?

(Also, I don't know who to ping for review of this PR as a whole.)

Comment (mppf) [reply compiler/dyno/include/chpl/types/ClassType.h:103]: Yes that change is good

**Comment by mppf** [comment]
> The last commit renames a method in compiler/dyno, which I haven't learned how to test yet. 

`make test-dyno`

**Pull request merged**